src/vs/workbench/common/editor.ts

```diff
@@ -22,6 +22,7 @@ import { IPathData } from 'vs/platform/windows/common/windows';
 import { coalesce } from 'vs/base/common/arrays';
 
 export const ActiveEditorContext = new RawContextKey<string | null>('activeEditor', null);
+export const CanSaveActiveEditorContext = new RawContextKey<boolean>('canSaveActiveEditor', false);
```

Comment:
@sbatten I am not 100% sure this is the best name. Checking https://code.visualstudio.com/docs/getstarted/keybindings#_when-clause-contexts it seems to me we would want the thing first ("active editor"). How about `activeEditorIsSaveable` or `activeEditorCanSave`?